app/models/search_eds.rb

```diff
@@ -17,11 +17,16 @@ class SearchEds
   RESULTS_PER_BOX = ENV['RESULTS_PER_BOX'] || 5
 
   def initialize
+    @attempts = 0
```

Comment:
We instantiate a new object for each request so... I'm going with I think so? If we reused the search objects this would be awful for sure. But since we create a new object on each search I think we're good:
https://github.com/MITLibraries/bento/blob/master/app/controllers/search_controller.rb#L68